### ISSUE TITLE
Training time much slower after "Learning rate dropout" commmit

### ISSUE THREAD

**Issue by TooMuchFun** [iperov/DeepFaceLab#519]
THIS IS NOT TECH SUPPORT FOR NEWBIE FAKERS
POST ONLY ISSUES RELATED TO BUGS OR CODE

## Expected behavior

Training time should remain similar after changes to model.

## Actual behavior

Training iteration length time has gone from _[#084910][1770ms][0.1957][0.1707]_ to _[#084944][2372ms][0.1674][0.0967]_ using same SAEHD settings.

When `sot` color transfer is enabled, iter time goes up 1000ms more to _[#084919][4131ms][0.1448][0.1058]_

## Steps to reproduce

Run large SAEHD model and profile iteration time. Enable color transfers to see train time start going very slow.

## Other relevant information
On in Google Colab on Tesla P100-PCIE-16GB instance.
`!python DeepFaceLab/main.py train --no-preview --training-data-src-dir $src_dir --training-data-dst-dir $dst_dir --model-dir $model_dir --model SAEHD`

> ==------------- Model Options --------------==
==                                          ==
==        sort_by_yaw: False                ==
==        random_flip: True                 ==
==         resolution: 192                  ==
==          face_type: f                    ==
==         learn_mask: False                ==
==     optimizer_mode: 2                    ==
==              archi: df                   ==
==            ae_dims: 512                  ==
==         ed_ch_dims: 21                   ==
==        random_warp: False                ==
== true_face_training: True                 ==
==   face_style_power: 0.0                  ==
==     bg_style_power: 0.0                  ==
==            ct_mode: none                 ==
==           clipgrad: True                 ==
==         batch_size: 10                   ==
==                                          ==
==--------------- Running On ---------------==
==                                          ==
==       Device index: 0                    ==
==               Name: Tesla P100-PCIE-16GB ==
==               VRAM: 16.00GB              ==
==                                          ==
==============================================

**Comment by TooMuchFun** [comment]
Maybe make as a config option at default 0.3? I can make PR

**Comment by iperov** [comment]
no thx.
Aim of SAEHD is provide the best possible deepfake quality in 2020.
Just reduce ae_dims to 256, and increase ed_ch a little.
256 works fine
